src/components/networked.js

```diff
@@ -209,6 +209,38 @@ AFRAME.registerComponent('networked', {
     this.el.sceneEl.systems.networked.register(this);
   },
 
+  update: function(oldData) {
+    const data = this.data;
+
+    // If owner has changed
+    if (oldData.owner !== data.owner) {
+      const wasMine = oldData.owner === NAF.clientId;
+      const isMine = this.isMine();
+
+      if (isMine && !wasMine) { // Gained ownership
+        this.removeLerp();
+        this.syncAll(undefined, true);
+        this.onOwnershipGainedEvent.oldOwner = oldData.owner;
+        this.el.emit(this.OWNERSHIP_GAINED, this.onOwnershipGainedEvent);
+      } else if (!isMine && wasMine) { // Lost ownership
+        this.onOwnershipLostEvent.newOwner = data.owner;
+        this.el.emit(this.OWNERSHIP_LOST, this.onOwnershipLostEvent);
```

Comment:
Not sure about all the changes here. We already have the some OWNERSHIP_LOST event emitted in networkUpdate. Is is emitted twice now?